ariadne_jwt/utils.py

```diff
@@ -70,9 +69,10 @@ def get_authorization_header(request):
 
 
 def get_payload(token, context=None):
+    ExpiredSignatureError = jwt.ExpiredSignatureError if 'ExpiredSignatureError' in dir(jwt.exceptions) else jwt.ExpiredSignature
```

Comment:
This is not required as the older version had ExpiredSignatureError aliased with ExpiredSignature 
```suggestion
```